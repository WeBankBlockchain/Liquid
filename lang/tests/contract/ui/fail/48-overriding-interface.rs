use liquid_lang as liquid;

#[liquid::interface(name = auto)]
mod foo {
    extern "liquid" {
        fn foo(&self);
        #[liquid(mock_context_getter = "another_getter")]
        fn foo(&self, i: i8);
    }
}

fn main() {}