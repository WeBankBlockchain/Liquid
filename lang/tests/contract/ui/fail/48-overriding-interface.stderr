error: overriding methods is not supported in liquid
 --> $DIR/48-overriding-interface.rs:6:9
  |
6 |         fn foo(&self);
  |         ^^^^^^^^^^^^^^
